src/security/Handshake.cc

```diff
@@ -631,10 +639,11 @@ Security::HandshakeParser::parseSupportedVersionsExtension(const SBuf &extension
 void
 Security::HandshakeParser::skipMessage(const char *description)
 {
-    // tkMessages/fragments can only contain messages of the same ContentType.
+    // tkMessages can only contain messages of the same ContentType.
     // To skip a message, we can and should skip everything we have [left]. If
     // we have partial messages, debugging will mislead about their boundaries.
```

Comment:
I think that "debugging will mislead" comment is stale because all skipMessage() callers add `foo [fragment]` to the message foo description. We are skipping message(s) and/or message fragments here.

```suggestion
    // we buffered a partial message, we will need to read/skip multiple times.
```

Reply:
Done.